### ISSUE TITLE
(WCAG 2.1) Repeated enumerator inputs and labels should use different IDs

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3565]
Required to meet WCAG 2.1 guidelines. Identified during an accessibility audit against WCAG 2.1.

**WCAG SC**: [4.1.1. Parsing](https://www.w3.org/TR/WCAG21/#parsing)

**Observed results**: When adding more enumerators, all of the label+input pairs that are created share the same ID. They should be unique.

Note: Currently, the labels are always the same anyway so there is no user impact. However, this will change with #2624.

**How to fix**: Newly added enumerators should generate a new unique ID.

![image](https://user-images.githubusercontent.com/9094240/192911407-c1c08224-a50f-422f-97e5-44abb39e189d.png)
